tweetcloud/app.py

```diff
@@ -1,91 +1,48 @@
-import argparse
-import logging
 import os
-import shutil
-from datetime import datetime, timedelta
+import tempfile
 
-import coloredlogs
+import click
 import twitter
 
 from tweetcloud import helpers
 
 
-def main():
-
-    parser = argparse.ArgumentParser(
-        description="Gets a dictionary of the most used words per week from an account"
-    )
-    parser.add_argument("screen_name", help="screen name to map")
-    parser.add_argument(
-        "number_of_tweets", type=int, help="number of tweets to grab from Twitter"
-    )
-
-    args = parser.parse_args()
-
-    if args.number_of_tweets > 3000:
-        coloredlogs.install()
-        logging.warning(
-            "TweetCloud does not currently support getting more than 3000 tweets, "
-            "so only 3000 tweets will be used for your gif..."
-        )
-
-    # Create the temp folder for the images
-    tmp_image_folder = "tmpPhotos"
-
-    # Create folder for intermediate images
-    if not os.path.isdir(tmp_image_folder):
-        os.mkdir(tmp_image_folder)
-
+@click.command()
+@click.option("-n", "--number-of-tweets", type=click.IntRange(0, 3000), default=1000)
```

Comment:
this errors if you pass `-n 4000`, but not in color. We can make custom (colored) errors by defining a custom type, but I thought we can get to that later.